lib/system/web/js/intersection_observer.js

```diff
@@ -0,0 +1,74 @@
+// ==============
```

Comment:
There should be docs on top of this file describing shortly what it is for. I realize the docs are not present in `resize_observer` - my fault, they should be there as well.